frontend/src/app/pages/marker-tutorial-page/marker-tutorial-page.component.html

```diff
@@ -10,7 +10,7 @@
                 <mat-card-subtitle>Step 1</mat-card-subtitle>
                 <p>First, choose dataset of the data you want to label. After that you will be taken to marker
                     page.</p>
-                <button class="nextPage" mat-raised-button matStepperNext color="primary">Next
+                <button  class="nextPage" mat-raised-button matStepperNext color="primary" data-cy="next1">Next
```

Comment:
Don't you think that `next1` are a little too cryptic? Maybe we should change it to something like `next1-marker-tutorial` to indicate that this is part of marker tutorial page? I am just wondering what would happen if we would like to have another button on different that would also need to be called `next1`?

Reply:
Tags don't have to be unique globally so I don't think it will be problem